package.json

```diff
@@ -26,7 +28,8 @@
     },
     "devDependencies": {
         "jasmine-node": "1.14.5",
-        "rewire": ">=2.1.3"
+        "rewire": ">=2.1.3",
+        "node-forge": "~0.6.18"
```

Comment:
That is unfortunate. The goal is to only run before publishing a new release. installing from npm is not intended to run this. We really do not want to impose node-forge as a dependency. 
Do you have a recommendation? Is there any scenario where --production is required to work? If so, perhaps I can check to see if node-forge is installed from the script and skip the certificate creation.